WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionSummaryViewModel.cs

```diff
@@ -55,9 +58,11 @@ public void UpdateTransaction(BuildTransactionResult transactionResult, Transact
 
 		var destinationAmount = _transaction.CalculateDestinationAmount();
 		AmountText = $"{destinationAmount.ToFormattedString()} BTC";
+		Amount = destinationAmount.ToDecimal(MoneyUnit.BTC);
 
 		var fee = _transaction.Fee;
 		FeeText = fee.ToFeeDisplayUnitFormattedString();
+		Fee = _transaction.Fee.ToDecimal(MoneyUnit.BTC);
```

Comment:
Doesn't this disregard the `FeeDisplayFormat` setting? Like if the user prefers sats it will still copy BTC.

Reply:
Oh, yes. Thanks for the catch! Fixed.